lmfdb/backend/searchtable.py

```diff
@@ -1492,3 +1492,7 @@ def count_distinct(self, col, query={}, record=True):
         - ``record`` -- (default True) whether to record the number of results in the stats table.
         """
         return self.stats.count_distinct(col, query, record=record)
+
+    def sum(self, col):
```

Comment:
This should take a constraint as an optional argument.  See `max` for an example.

Reply:
You should probably change `sum_column` rather than add a new `sum` function.

Reply:
optional `constraint` now added. I actually renamed `sum_column` (something i put in before) to `sum` to be more in line with `min` and `max` naming.